### PR TITLE
Add missing input keys for events

### LINKED ISSUE #3392: NumpadReturn key isn't listed in the auto complete
## Describe the bug
`NumpadReturn` with keycode `3013` isn't visible in the autocomplete for some reason.

image

## Other details
* 5.0.124
* Web app
* Desktop
* W10

### PR DESCRIPTION
**Pull request by ClementPasteau**
Fixes #3392 

I've copied the keys being handled in `inputtools.ts` and reordered them as well to be the same.

image

There are 3 keys suggested in the autocomplete that don't seem handled by the engine: `LBracket`, `RBracket`, `Semicolon`
I'd be tempted to remove them, but also wary I'm missing something.

cc @Bouh

### CODE REVIEW

File: newIDE/app/src/EventsSheet/ParameterFields/KeyField.js [hunk 1]
```diff
@@ -95,12 +105,9 @@ const keyNames = [
...
   'Pause',
-  'RControl',
-  'LControl',
-  'RAlt',
-  'LAlt',
-  'RShift',
-  'LShift',
+  'LBracket', // seems not working
+  'RBracket', // seems not working
+  'SemiColon', // seems not working
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/ParameterFields/KeyField.js:110]: Semicolon seems working according to https://keycode.info/ (keycode 186) (notice the lowercase c). Might be something we need to add in the game engine?

**Review by 4ian — APPROVED** [review]
Approving this to go ahead after checking the semicolon :)

File: GDJS/Runtime/events-tools/inputtools.ts [hunk 2]
```diff
@@ -68,18 +68,18 @@ namespace gdjs {
         RAlt: 2018,
         LSystem: 1091,
         RSystem: 2091,
-        /*"Menu": sf::Keyboard::Menu ,
```
Comment (ClementPasteau) [inline GDJS/Runtime/events-tools/inputtools.ts:71]: I went back to 2013 to understand why those values had been commented, but looks like they were added commented from the beginning :)
Something I'm missing or can we add them without risk?

Comment (4ian) [reply GDJS/Runtime/events-tools/inputtools.ts:71]: Can be added without risk.

Comment (4ian) [reply GDJS/Runtime/events-tools/inputtools.ts:71]: Can be added without risk.

Comment (4ian) [reply GDJS/Runtime/events-tools/inputtools.ts:71]: Can be added without risk!

Comment (4ian) [reply GDJS/Runtime/events-tools/inputtools.ts:71]: Can be added without risk!

**Pull request merged**